src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,27 @@
+@media (min-width: 992px) {
+    .mobile-hide {
+        display: initial;
+    }
+}
+
+.mobile-hide {
+    display: none !important;
```

Comment:
Optional: Ideally, this CSS should be in the story row component applied on the entire component since this only needs to be hidden on mobile in the context of it being a column in the story row.